test/users/ferguson/histogram/reductionex_class.future

```diff
@@ -1,7 +1,3 @@
-bug: nested type specifier does not work for reduction
+language: do we want support for referring to enclosing types' fields?
```

Comment:
Language is one of the "area" tags for the repo, but we want to be using the "type" tags for future categorization.  In this case, I think "design" is appropriate.